bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/azure/AzureReplicatedBagVerifier.scala

```diff
@@ -37,6 +41,17 @@ class AzureReplicatedBagVerifier(
     location: AzureBlobLocation
   ): String =
     location.name.replace(root.namePrefix, "")
+
+  override def isRetriable(error: BagVerifierError): Boolean = error match {
+    // When reading from Azure, sometimes we see errors like
+    // "StoreReadError(reactor.core.Exceptions$ReactiveException: java.util.concurrent.TimeoutException: Did not
+    // observe any item or terminal signal within 60000ms in 'map' (and no fallback has been configured))"
+    // These are retriable errors so let's mark them as such
+    case BagVerifierError(e: RuntimeException, _)
+        if e.getMessage.contains("TimeoutException") =>
+      true
+    case _ => false
+  }
```

Comment:
Minor: since the second part of a BagVerifierError is always going to be a user-facing message, maybe match over `error.e` instead? i.e.

```scala
error.e match {
  case e: RuntimeException if e.getMessage.contains("TimeoutException") => true
  case _ => false
}
```